.clang-format

```diff
@@ -62,7 +62,7 @@ IncludeCategories:
     Priority:        4
 
   # 5. Pixar + USD headers
-  - Regex:           '^<pxr/'
+  - Regex:           '(pxr_python\.h|^<pxr/)'
```

Comment:
Sort this new top-level header in with the other pxr headers. I would have preferred if it was at the top of the group, but clang-format puts it at the bottom.